packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -579,18 +601,17 @@ export async function createHotReloaderTurbopack(
         const turbopackConnected: TurbopackConnectedAction = {
           action: HMR_ACTIONS_SENT_TO_BROWSER.TURBOPACK_CONNECTED,
         }
-        client.send(JSON.stringify(turbopackConnected))
+        sendToClient(client, turbopackConnected)
 
         const errors: CompilationError[] = []
-        const addIssues = (entryIssues: Map<string, Issue>) => {
+
+        for (const entryIssues of currentEntryIssues.values()) {
           for (const issue of entryIssues.values()) {
             errors.push({
               message: formatIssue(issue),
             })
           }
         }
-        clientIssues.forEach(addIssues)
-        currentEntryIssues.forEach(addIssues)
 
```

Comment:
I think that was an incorrect rebase. It should also add issues from `clientIssues`

Reply:
`clientIssues` can't have any entries at this point?

Reply:
entries?

`clientIssues` are issues from subscriptions of that connection/client. They should also be send via websocket.

Reply:
Yes, the map can't have any entries at this point, it's empty because the client just connected